### ISSUE TITLE
Manga109 PSNR x3 Y score?

### ISSUE THREAD

**Issue by jcai20** [sanghyun-son/EDSR-PyTorch#313]
Anyone did experiemnt on the Manga109 dataset (x3 and Y_PSNR)?

**Comment by nan-rock** [comment]
i did, but i am facing problem for all the datasets for scale factor 3. My validation accuracy on div2k dataset is coming good, but the testing accuracy for all the datasets is coming very poor. Could u please help me out
